alias/gui.h

```diff
@@ -27,12 +27,15 @@
 
 struct Alias;
 
+// clang-format off
```

Comment:
We don't need these in header files.
I routinely format the `.c` files, but the header files are tidied by hand (and vim-fu).

Reply:
I use Emacs, and I configured it to clang-format on save.
I guess I have to define a workaround :-)